examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -153,7 +150,19 @@
 )
 
 parser.add_argument(
-    "--enable_pin_memory", action="store_true", help="Enabled the pin_memory feature of Pytroch's DataLoader",
+    "--enable_pin_memory", action="store_true", help="Enables the pin_memory feature of Pytroch's DataLoader",
+)
+
+parser.add_argument(
+    "--state_tracker", type=str, default='baseline', help="Specifies the state tracker mode",
+)
```

Comment:
let's add supported choices here, I'll add mine, could you add options you're using

Reply:
Done

Reply:
Done